jbmc/regression/jbmc-concurrency/get-current-thread/A.java

```diff
@@ -0,0 +1,101 @@
+import java.lang.Thread;
+import org.cprover.CProver;
+
+public class A
+{
+  public static int g;
+
+  // expected verification success
+  public void me()
+  {
+    int g = CProver.getCurrentThreadID();
+    assert(g == 0);
+  }
+
+  // expected verification success
+  // --
+  // KNOWNBUG
+  public void me_bug()
+  {
+    CProver.startThread(333);
+    int g = 1;
+    assert(g == 1);
+    CProver.endThread(333);
+  }
+
+  // expected verification success
+  // --
+  // KNOWNBUG: see me_bug()
```

Comment:
I don't understand that comment, at `me_bug` there is also no real description

Reply:
added description